Dockerfile.release

```diff
@@ -1,15 +1,19 @@
 FROM alpine:3.12
-
-ARG BINARY=nri-prometheus
+ARG TARGETOS
+ARG TARGETARCH
+ARG TINI_VERSION=v0.19.0
 
 RUN apk add --no-cache --upgrade \
-        ca-certificates
+        ca-certificates \
+        curl
 
-COPY ${BINARY} /bin/nri-prometheus
+RUN curl -fsSLo /sbin/tini "https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-static-${TARGETARCH}" && \
+    chmod +x /sbin/tini
```

Comment:
I'd rather `apk add tini` instead.

Reply:
I am doing this since I needed the static tini version, when installing the classical one it was failing due to missing DDLS in arm64

Reply:
I'll test it again, it it possible that it was failing since it was fetching the wrong image at first